documents/decorators.py

```diff
@@ -0,0 +1,19 @@
+from flask import jsonify, request
+import documents.state_machine as state_machine
+
+def auth(func):
```

Comment:
what does 'auth' mean? Does it do the authorization, or check someone else's authorization? 
Rename accordingly